cloudevents/working-drafts/xml-format.md

```diff
@@ -130,15 +130,17 @@ Example:
 
 ### 3.3 XML Data
 
-XML data MUST be carried in an element with a defined type of `xs:any`
-allowing a single child XML element (with any required namespace definitions)
-to be represented.
+XML data MUST be carried in an element with a defined type of `xs:any` with
+a single child XML element (with any required namespace definitions).
+
+The child business data XML elements MUST NOT reside in the CloudEvent namespace,
```

Comment:
I'd like us to discuss this in the next WG meeting where both Jem and I are present.

Pros:
- Avoids *accidentally* using the CloudEvent namespace

Cons:
- May prevent genuinely useful functionality, e.g. "wrap a batch as a payload in a single CloudEvent for transports that don't understand batches".

Reply:
the phrase "child business data" might be hard for some people to parse. Perhaps just "The child XML elements MUST NOT...".

However, I actually do think the CE namespace is ok if the data is a nested structured CloudEvent, no?

Reply:
I *will* relax this language - i'm not sure how we ended up here but having this restriction means that you can't (very easily) wrap an XML CloudEvent inside another XML CloudEvent.

This "Russian Doll" model is a useful construct to support.